config/plugins/visualizations/charts/static/repository/visualizations/ngl/viewer/wrapper.js

```diff
@@ -0,0 +1,42 @@
+define( [ 'utilities/utils', "plugins/ngl/viewer" ], function( Utils, ngl ) {
+    return Backbone.Model.extend({
+        initialize: function( options ) {
+            var dataset = options.dataset,
+                settings = options.chart.settings,
+                stage = new ngl.Stage( options.targets[ 0 ], { backgroundColor: settings.get( 'backcolor' ) } ),
+                viewer_options = {},
+                representation_parameters = {},
+                stage_parameters = {};
+
+            _.each( settings.attributes, function( value, key ) {
+                if ( key.startsWith( 'viewer|' ) ) {
+                    viewer_options[ key.replace( 'viewer|', '' ) ] = value;
+                }
+            } );
+            representation_parameters = { 
+                radius: settings.get( 'radius' ),
+                scale: settings.get( 'scale' ),
+                assembly: settings.get( 'assembly' ),
+                color: settings.get( 'colorscheme' ),
+                opacity: settings.get( 'opacity' )
+            };
+            stage_parameters = { ext: dataset.extension, name: dataset.name, defaultRepresentation: true };
```

Comment:
Cool, good to know.  Since it's unused (and not required), let's cut that out, and then I'm +1 and will merge?